app/components/govuk_component/back_link.html.erb

```diff
@@ -0,0 +1,3 @@
+<a href="<%= @href %>" class="govuk-back-link<% if @classes.present? %> <%= @classes%><% end %>">
```

Comment:
I would like to extend this to allow additional html attributes.

Reply:
Additional html attributes can now be added by using the optional `attributes` params